src/clue.js

```diff
@@ -1,94 +1,226 @@
+    age:          31,
+    image:        "https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg",
+    occupation:   "Actor"
+  }
+  
+  var mrsPeacock = {
+    first_name:   "Eleanor",
+    last_name:    "Peacock",
+    color:        "blue",
+    description:  "She is from a wealthy family and uses her status and money to earn popularity",
+    age:          36,
+    image:        "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+    occupation:   "Socialité"
+  }
+  
+  var mrMustard = {
+    first_name:   "Jack",
+    last_name:    "Mustard",
+    color:        "yellow",
+    description:  "He is a former football player who tries to get by on his former glory",
+    age:          62,
+    image:        "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+    occupation:   "Retired Football player"
+  }
+
+charactersArray.push(mrMustard, mrsPeacock, missScarlet, profPlum, drOrchid, mrGreen);
+
+
+////////////////////////////////// Rooms Collection
+
 var roomsArray = [];
 
-// Weapons Collection
+var diningRoom = {
+  name: "Dining Room",
+}
+
+var conservatory = {
+  name: "Conservatory",
+}
+
+var kitchen = {
+  name: "Kitchen",
+}
+
+var study = {
+ name: "Study",
+}
+
+var library = {
+ name: "Library",
+}
+var billiardRoom = {
+ name: "Billiard Room",
+}
+
+var lounge = {
+ name: "Lounge",
+}
+
+var ballroom = {
+ name: "Ballroom",
+}
+
+var hall = {
+ name: "Hall",
+}
+
+var spa = {
+ name: "Spa",
+}
+
+var livingRoom = {
+ name: "Living Room",
+}
+
+var observatory = {
+ name: "Observatory",
+}
+
+var theater = {
+ name: "Theater",
+}
+
+var guestHouse = {
+ name: "Guest House",
+}
+
+var patio = {
+ name: "Patio",
+}
+
+roomsArray.push(patio, guestHouse, theater, observatory, livingRoom, spa, hall, ballroom, lounge, billiardRoom, library, study, kitchen, conservatory, diningRoom);
+
+////////////////////////////// Weapons Collection
+
 var weaponsArray = [];
+
+var rope = { 
+  name: "rope",
+  weight: 10
+}   
+
+var knife = {
+  name: "knife", 
+  weight: 8
+}
+
+var candlestick = {
+  name: "candlestick", 
+  weight: 2
+}
+
+var dumbbell = {
+  name: "dumbbell", 
+  weight: 30
+}
+
+var poison = {
+  name: "poison",
+  weight: 2
+}
+
+var axe = {
+  name: "axe",
+  weight: 15
+}
+
+var bat = {
+  name: "bat",
+  weight: 13
+}
+
+var trophy = {
+  name: "trophy",
+  weight: 25
+}
+
+var pistol = {
+  name: "pistol",
+  weight: 20
+}
+
+weaponsArray.push(rope, knife, candlestick, dumbbell, poison, axe, bat, trophy, pistol);
+
+//// Random Selector
+
+var randomSelector = (arr) =>  {
+  var randomIndex = Math.floor(Math.random() * arr.length);
+  if(arr.length !== 0) {
+    var randomElement = arr[randomIndex];
+  }
+  return randomElement;
+};
+
+//// Pick Mystery
+
+var mysteryCards = [];
+
+  var mysteryCharacter = randomSelector(charactersArray);
+  var mysteryWeapon = randomSelector(weaponsArray);
+  var mysteryRoom = randomSelector(roomsArray);
+
+function pickMistery(){
```

Comment:
Try to be consistent with the format. If you put a space between " ) " and " { " on the functions which I thing is correct. Put it always